tests/phpunit/RetryTest.php

```diff
@@ -456,9 +459,13 @@ public function testRetryNetworkErrorExecute(): void
 
         $extractor = new Common($config['parameters'], [], $logger);
         // plant the tainted PDO into the extractor
-        $reflectionProperty = new \ReflectionProperty($extractor, 'db');
-        $reflectionProperty->setAccessible(true);
-        $reflectionProperty->setValue($extractor, $this->taintedPdo);
+        $connectionProperty = new \ReflectionProperty($extractor, 'connection');
```

Comment:
Tieto testy su dost problematicke, lebo modifikuju interne hodnoty objektov, ... ale ponechal som ich a fixol.
V buducnosti ich bude potrebne zrefactorovat. 
Ide o to dostat do objektov modifikovanu verziu `PDO`, ktora v istej faze vyvola vypnutie MySQL servera, ... a testuje sa napr. ze nastane retry, alebo sa vyhodi spravna chyba.

Reply:
jo, tyto testy jsou picoviny

Reply:
i'm thinking that maybe we should remove this test class for now, and add it in after the new major version is master?